lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -389,24 +405,35 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
       // attribute which specifies the internal path into the extern module.
       // This string attribute will be used to generate the final xmr.
       auto internalPaths = extRefMod.getInternalPaths();
-      // No internalPaths implies no RefType ports.
-      if (internalPaths.empty())
-        return success();
+      // Set internalPaths to macro?
```

Comment:
Fix this comment, leftover from thinking of populating `internalPaths` with ABI information first (pre-pass?) so this can just use it blindly as current IR model says ref resolution information is stored there.